### ISSUE TITLE
Trident acetabulum 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7643]
**Preferred term label:**
Trident acetabulum 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
near - Acetabular spurs HP:0010454

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617405 
PMID: 26044572  5/7  (does this replace acetablar spurs?)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added. Subclass of acetabular spurs
